mllm/backends/ascend/AscendDispatcher.hpp

```
@@ -0,0 +1,41 @@
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <memory>
+
+#include "mllm/engine/Dispatcher.hpp"
+#include "mllm/utils/Common.hpp"
+
+namespace mllm::ascend {
+
+struct AscendDispatcherOptions {
```

Clang (14.0.6)

[error] constructor does not initialize these fields: MLLM_EMPTY_SCOPE

(cppcoreguidelines-pro-type-member-init,-warnings-as-errors)